src/common/node/childProcess.ts

```diff
@@ -55,44 +61,32 @@ export class ChildProcess {
         return this.exec(command, options).outcome.then(stdout => stdout.toString());
     }
 
-    public spawnWithExitHandler(command: string, args?: string[], options: ISpawnOptions = {}): ISpawnResult {
+    public spawn(info: ISpawnInfo, options: ISpawnOptions = {}): ISpawnResult {
```

Comment:
I think we should leave this signature to expect:
command: string, args?: string[], options: ISpawnOptions = {}.
If you *need* to add an option, I'd modify options to be { customOptions: something, spawnOptions: something }